### PR TITLE
closes #9 added scripts to run test.

### LINKED ISSUE #9: Create scripts that make it easier to run stress test
A script to run end to end stress test AND scripts to run individual stess command would be make running test much easier.

### PR DESCRIPTION
**Pull request by keith-turner**
Scripts use new classpath command, so no longer building shaded jar.

### CODE REVIEW

**Comment by mikewalch** [comment]
The unit tests failing due to the exception below:

```
java.lang.NoSuchMethodError: org.apache.accumulo.core.volume.VolumeConfiguration.getVolumeUris(Lorg/apache/accumulo/core/conf/AccumuloConfiguration;Lorg/apache/hadoop/conf/Configuration;)[Ljava/lang/String;
  at org.apache.accumulo.server.fs.VolumeManagerImpl.get(VolumeManagerImpl.java:411)
  at org.apache.accumulo.server.fs.VolumeManagerImpl.get(VolumeManagerImpl.java:404)
  at org.apache.accumulo.server.init.Initialize.main(Initialize.java:574)
  at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
  at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  at java.lang.reflect.Method.invoke(Method.java:606)
  at org.apache.accumulo.start.Main$1.run(Main.java:141)
  at java.lang.Thread.run(Thread.java:745)
```

File: bin/configure-fluo.sh [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+BIN_DIR=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
+
+if [ ! -f $BIN_DIR/../conf/env.sh ] 
+then
+  . $BIN_DIR/../conf/env.sh.example
+else
+  . $BIN_DIR/../conf/env.sh
+fi
+
+if [ "$BUILD" = "true" ]
```
Comment (mikewalch) [inline bin/configure-fluo.sh:8]: it doesn't look like this repo is used anywhere else in the code...

File: bin/configure-fluo.sh [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+then
+  #install fluo
+  #TODO can remove when fluo beta is released
+  (cd $BIN_DIR/..; mkdir -p git; cd git; git clone https://github.com/fluo-io/fluo.git; cd fluo; mvn install -DskipTests -Dfindbugs.skip=true)
+  #build fluo-stress
+  (cd $BIN_DIR/..;mvn package -DskipTests)
+fi
+
+#copy stess jar
+cp $STRESS_JAR $FLUO_HOME/lib/observers
```
Comment (mikewalch) [inline bin/configure-fluo.sh:24]: should check to make sure `$FLUO_HOME` is set before using it

Comment (mikewalch) [inline bin/configure-fluo.sh:24]: could make sure `$STRESS_JAR` exists like in `run-test.sh`

File: conf/env.sh.example [hunk 3]
```diff
@@ -0,0 +1,19 @@
+#configuration for all scripts
```
Comment (mikewalch) [inline conf/env.sh.example:5]: While you mention this in the README, you could also mention below that you can set `FLUO_HOME` in env.sh:

```
# Uncomment and set if FLUO_HOME is not set in your .bashrc
#FLUO_HOME=/path/to/fluo-install
```

File: conf/env.sh.example [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+export HADOOP_CLASSPATH=`$FLUO_HOME/bin/fluo classpath -az`
+LIB_JARS=`$FLUO_HOME/bin/fluo classpath -azl`
+STRESS_JAR=$BIN_DIR/../target/fluo-stress-0.0.1-SNAPSHOT.jar
+FLUO_PROPS=$FLUO_HOME/conf/fluo.properties
+
+#configuration for run-test.sh follows
```
Comment (mikewalch) [inline conf/env.sh.example:16]: Could put comments describing each configuration item

File: README.md [hunk 5]
```diff
@@ -66,138 +80,106 @@ These tests can be run using `mvn verify`.
...
-```
+The [bin directory](/bin) contains a set of scripts to help run this test on a
+cluster.  These scripts make the following assumpitions.
 
-Next, build the module:
-```
-mvn package assembly:single
-```
+ * `FLUO_HOME` environment variable is set.  If not set, then set it in `conf/env.sh`.
+ * Hadoop `yarn` command is on path.
```
Comment (mikewalch) [inline README.md:87]: Could also check for the existence of these commands in the scripts.  See http://stackoverflow.com/questions/592620/check-if-a-program-exists-from-a-bash-script

**Comment by mikewalch** [comment]
A lot of the verification I mentioned could be put in a `load-env.sh` script which loads env.sh and verifies that everything is set up correctly.  This way all of the scripts could use it.

**Comment by keith-turner** [comment]
There is a mixing of accumulo versions, thats probably causing test to fail

```
$ mvn dependency:tree | grep accumulo
[INFO] |  +- io.fluo:fluo-accumulo:jar:1.0.0-beta-1-SNAPSHOT:compile
[INFO] |  +- org.apache.accumulo:accumulo-core:jar:1.6.1:compile
[INFO] +- org.apache.accumulo:accumulo-minicluster:jar:1.6.2:test
[INFO] |  +- org.apache.accumulo:accumulo-gc:jar:1.6.2:test
[INFO] |  +- org.apache.accumulo:accumulo-master:jar:1.6.2:test
[INFO] |  +- org.apache.accumulo:accumulo-monitor:jar:1.6.2:test
[INFO] |  +- org.apache.accumulo:accumulo-server-base:jar:1.6.2:test
[INFO] |  +- org.apache.accumulo:accumulo-start:jar:1.6.2:compile
[INFO] |  +- org.apache.accumulo:accumulo-tracer:jar:1.6.2:test
[INFO] |  +- org.apache.accumulo:accumulo-tserver:jar:1.6.2:test
[INFO] +- org.apache.accumulo:accumulo-test:jar:1.6.2:test
[INFO] |  +- org.apache.accumulo:accumulo-fate:jar:1.6.2:compile
[INFO] |  +- org.apache.accumulo:accumulo-native:tar.gz:1.6.2:test
[INFO] |  +- org.apache.accumulo:accumulo-trace:jar:1.6.2:compile
```

File: bin/configure-fluo.sh (hunk 2 above)
Comment (keith-turner) [inline bin/configure-fluo.sh:24]: fixed in c8849f9

File: bin/configure-fluo.sh (hunk 1 above)
Comment (keith-turner) [inline bin/configure-fluo.sh:8]: fixed in c8849f9

File: conf/env.sh.example (hunk 3 above)
Comment (keith-turner) [inline conf/env.sh.example:5]: fixed in c8849f9

File: conf/env.sh.example (hunk 4 above)
Comment (keith-turner) [inline conf/env.sh.example:16]: fixed in c8849f9

File: README.md (hunk 5 above)
Comment (keith-turner) [inline README.md:87]: fixed in c8849f9

**Comment by keith-turner** [comment]
I added a dependency management that specifies a version for accumulo core to fix the unit test problem.  After fixing that, I ran into fluo-io/fluo#444

**Comment by mikewalch** [comment]
+1

**Pull request merged**